po/fr.po

```diff
@@ -238,6 +238,354 @@ msgstr "Pacman est en cours d'utilisation, veuillez patienter..."
+msgid "    paru {-D --database}    <options> <package(s)>"
+msgstr "    paru {-D --database}    <options> <paquet(s)>"
+
+#: src/help.rs:13
+msgid "    paru {-F --files}       [options] [package(s)]"
+msgstr "    paru {-F --files}       [options] [paquet(s)]"
+
+#: src/help.rs:14
+msgid "    paru {-Q --query}       [options] [package(s)]"
+msgstr "    paru {-Q --query}       [options] [paquet(s)]"
+
+#: src/help.rs:15
+msgid "    paru {-R --remove}      [options] <package(s)>"
+msgstr "    paru {-R --remove}      [options] <paquets(s)>"
+
+#: src/help.rs:16
+msgid "    paru {-S --sync}        [options] [package(s)]"
+msgstr "    paru {-S --sync}        [options] [paquets(s)]"
+
+#: src/help.rs:17
+msgid "    paru {-T --deptest}     [options] [package(s)]"
+msgstr "    paru {-T --deptest}     [options] [paquet(s)]"
+
+#: src/help.rs:18
+msgid "    paru {-U --upgrade}     [options] [file(s)]"
+msgstr "    paru {-U --upgrade}     [options] [fichier(s)]"
+
+#: src/help.rs:20
+msgid "New operations:"
+msgstr "Nouvelles opérations:"
+
+#: src/help.rs:21
+msgid "    paru {-P --show}        [options]"
+msgstr "    paru {-P --show}        [options]"
+
+#: src/help.rs:22
+msgid "    paru {-G --getpkgbuild} [package(s)]"
+msgstr "    paru {-G --getpkgbuild} [paquet(s)]"
+
+#: src/help.rs:24
+msgid "If no arguments are provided 'paru -Syu' will be performed"
+msgstr "Si aucun argument n'est fourni 'paru -Syu' sera exécuté"
+
+#: src/help.rs:26
+msgid "Options without operation:"
+msgstr "Options sans opérations:"
+
+#: src/help.rs:27
+msgid "    -c --clean            Remove unneeded dependencies"
+msgstr "    -c --clean             Supprime les dépendances inutiles"
+
+#: src/help.rs:28
+msgid ""
+"       --gendb            Generates development package DB used for updating"
+msgstr ""
+"       --gendb             Génère la BDD de paquets de développement "
+"utilisée pour mettre à jour"
+
+#: src/help.rs:30
+msgid "New options:"
+msgstr "Nouvelles options:"
+
+#: src/help.rs:31
+msgid "       --repo              Assume targets are from the repositories"
+msgstr "       --repo              Suppose que les cibles sont dans les depôts"
+
+#: src/help.rs:32
+msgid "    -a --aur               Assume targets are from the AUR"
+msgstr "    -a --aur               Suppose que les cibles sont dans l'AUR"
+
+#: src/help.rs:33
+msgid "    --aururl    <url>      Set an alternative AUR URL"
+msgstr "    --aururl    <url>      Défini une URL alternative pour l'AUR"
+
+#: src/help.rs:34
+msgid "    --clonedir  <dir>      Directory used to download and run PKGBUILDs"
+msgstr ""
+"    --clonedir  <dir>      Dossier utilisé pour télécharger et exécuter "
+"PKGBUILDs"
+
+#: src/help.rs:36
+msgid "    --makepkg   <file>     makepkg command to use"
+msgstr "    --makepkg   <fichier>      Commande makepkg à utiliser"
+
+#: src/help.rs:37
+msgid "    --mflags    <flags>    Pass arguments to makepkg"
+msgstr "    --mflags    <arguments>    Passe des arguments à makepkg"
```

Comment:
Could you shorten this to <args> so the text after can line up?

Reply:
Specifically <arguments> to <args>

Reply:
I checked and didn't find a shorter word. args is English not French, it will be understandable but a bit weird. If you really want it to be aligned I can change it

Reply:
Okay leave it as is then. But I don't really get why the same word can't be shortened.